src/analytics.ts

```diff
@@ -5,18 +5,26 @@ import { TCoreAttributes, TEvents } from './types'
 type Options = {
     growthbookKey?: string
     growthbookDecryptionKey?: string
-    rudderstackKey: string
+    rudderstackKey?: string
 }
 
 export function createAnalyticsInstance(options?: Options) {
     let _growthbook: Growthbook,
-        _rudderstack: RudderStack,
+        _rudderstack: RudderStack | null,
         core_data: Partial<TCoreAttributes> = {},
         tracking_config: { [key: string]: boolean } = {},
         offline_cache: { [key: string]: { event: keyof TEvents; payload: TEvents[keyof TEvents] } } = {}
 
     const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey }: Options) => {
-        _rudderstack = RudderStack.getRudderStackInstance(rudderstackKey)
+        // If Rudderstack is disabled in Firebase remote config
```

Comment:
When `rudderstackKey` is not passed, it means that Rudderstack is disabled in Firebase config. In core, its done by:
```tsx
   const { data } = useRemoteConfig();

    React.useEffect(() => {
        initDatadog(tracking_datadog);
        if (process.env.RUDDERSTACK_KEY) {
            const config = {
                ...
                rudderstackKey: data.tracking_rudderstack ? process.env.RUDDERSTACK_KEY : undefined,
            };
            Analytics.initialise(config);
        }

    }, [data]);
```